docs/src/main.css

```diff
@@ -378,7 +379,6 @@ body._hasBackground {
 .Prose code pre {
   font-family: 'Roboto Mono';
   background-color: transparent;
-  width: 100%;
   display: block;
 }
 
```

Comment:
These last changes _probably_ screw up some of my utility class docs (I'd made these changes to make the code blocks in that doc look right...)